frontend/ui/lib/pages/trip_view_page.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+import 'package:tripmeout/widgets/trip_view_widget.dart';
+
+class TripViewPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
```

Comment:
Consider using the [default app bar implementation](https://github.com/googleinterns/step23-2020/blob/49f8414179f0ff718fd65a4d263ee96ab61db031/frontend/ui/lib/widgets/default_app_bar.dart#L3) that @KoopaKing added.